integration_test/gce/gce_testing.go

```diff
@@ -1299,6 +1299,14 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 		}
 	}
 
+	// Pre-installed jupyter services on DLVM images cause port conflicts for third-party apps.
+	// See b/347107292.
+	if IsDLVMImage(vm.ImageSpec) {
+		if _, err := RunRemotely(ctx, logger, vm, "sudo service jupyter stop || true"); err != nil {
```

Comment:
curious, why do this here as opposed to in the install script for the affected apps? This only affects tomcat and jetty right?

Reply:
It could affect other apps or tests in the future (port 8080 is super common), and jupyter isn't really relevant to the tests to begin with, so I figure it's both safer and simpler to disable it outright.

The only downside I can think of is that failed tests are currently our only useful signal into which specific apps are problematic -- had we done it this way from the start, we likely wouldn't have known that tomcat and jetty were impacted, which reduces opportunity for e.g. pro-active documentation. I think this is still a reasonable trade-off.